shared/common-adapters/markdown.native.js

```diff
@@ -37,6 +37,9 @@ const codeSnippetBlockTextStyle = {
 
 const quoteBlockStyle = {borderLeftColor: globalColors.lightGrey2, borderLeftWidth: 3, paddingLeft: 8}
 
+// The Text component adds default styles which we need to unset so that
+// styles applied to Markdown parent take effect. For instance, we need
+// to unset the default color applied by so that works.
```

Comment:
Ack, GitHub markdown formatting ate part of my suggestion, here it was in full: "The Text component adds default styles which we need to unset so that styles applied to Markdown parent take effect. For instance, we need to unset the default color applied by `<Text type="body">` so that `<Markdown style={{color: ...}}>` works."

I think this example is useful for us to remember the context of this, could you please add it?